operations/outletitems.md

```diff
@@ -41,120 +44,118 @@ Note this operation uses [Pagination](../guidelines/pagination.md) and supports
 | `AccessToken` | string | required | Access token of the client application. |
 | `Client` | string | required | Name and version of the client application. |
 | `EnterpriseIds` | array of string | optional, max 1000 items | Unique identifiers of the [Enterprises](enterprises.md#enterprise). If not specified, the operation returns data for all enterprises within scope of the Access Token. |
+| `Ids` | array of string | optional, max 1000 items | Unique identifiers of the [Outlet items](#outlet-item). If not specified, the operation returns data for all [Outlet items](#outlet-item) within scope of the Access Token. |
 | `ConsumedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Outlet item](#outlet-item) was consumed. Required if no other filter is provided. |
-| `ClosedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Outlet bill](#outlet-bill) was closed. |
 | `UpdatedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Outlet bill](#outlet-bill) was updated. |
-| `Currency` | string | optional | ISO-4217 code of the [Currency](currencies.md#currency) the item costs should be converted to. |
-| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of outlet items returned. |
+| `ClosedUtc` | [Time interval](_objects.md#time-interval) | optional, max length 3 months | Interval in which the [Outlet bill](#outlet-bill) was closed. |
+| `Currency` | string | optional | ISO-4217 code of the [Currency](#currency) the item costs should be converted to. |
+| `Limitation` | [Limitation](../guidelines/pagination.md#limitation) | required | Limitation on the quantity of data returned. |
+| ~~`StartUtc`~~ | ~~string~~ | ~~optional~~ | **Deprecated!** |
+| ~~`EndUtc`~~ | ~~string~~ | ~~optional~~ | **Deprecated!** |
+| ~~`TimeFilter`~~ | ~~string~~ | ~~optional~~ | **Deprecated!** |
```

Comment:
issue: Since these were removed a long time ago and they're present on the request, I suggest hiding them from the documentation by putting `[InternalImplementation]` on them.